Modules/Remote/AdaptiveDenoising.remote.cmake

```diff
@@ -1,5 +1,5 @@
 itk_fetch_module(AdaptiveDenoising
-  "A spatially adaptive denoising image filter."
+  "A spatially adaptive denoising image filter using non-local means."
   GIT_REPOSITORY http://github.com/ntustison/ITKAdaptiveDenoising.git
```

Comment:
Can `.DS_Store` file be removed, and preferably added to `.gitignore`?

Reply:
Yep, sounds good.  Just an oversight on my part.